.env

```diff
@@ -0,0 +1,11 @@
+REACT_APP_AUTH_ENABLED=true
+# Oathkeeper endpoint
+REACT_APP_AUTH_API_BASE_URL=http://127.0.0.1:4455/.ory/kratos/public
+REACT_APP_AUTH_MOCK_API=false
+REACT_APP_REMOTE_API_BASE_URL=/remote
+REACT_APP_REMOTE_MOCK_API=true
+# Roles Microfrontend
+REMOTE_1_URL=http://localhost:3012
+# Transfers Microfrontend
+REMOTE_2_URL=http://localhost:3013
+DEV_PORT=3010
```

Comment:
`.env` file shouldn't be committed as it's intended to be used locally.
You can create `.env.examle` for the purpose of sample env vars.

https://dev.to/getd/how-to-manage-secrets-and-configs-using-dotenv-in-node-js-and-docker-2214

Reply:
Thought it would be fine since this repo handles any secrets but you're right. done. added `.env.example`